<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeracaoGrades</class>
 <widget class="QMainWindow" name="GeracaoGrades">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeracaoGrades</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="carregarPlanilias">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar Planilias</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gerarGrades">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerar Grade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gerarSaida">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerar Saida</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>801</width>
      <height>411</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>391</height>
     </size>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>133</number>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Turma</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Creditos</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addTurma">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>440</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar Turma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectAll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Marcar Tudo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unselectAll">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Desmarcar Tudo</string>
    </property>
   </widget>
   <zorder>carregarPlanilias</zorder>
   <zorder>gerarGrades</zorder>
   <zorder>gerarSaida</zorder>
   <zorder>addTurma</zorder>
   <zorder>tableWidget</zorder>
   <zorder>selectAll</zorder>
   <zorder>unselectAll</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
